themes/bootstrap3/less/components/record.less

```diff
@@ -353,3 +353,10 @@
   align-items: center;
   justify-content: center;
 }
+
+.backtoorigin > p > a {
```

Comment:
I don't think it makes sense to repeat these styles in both the search and record components, since all the components get merged together in the compilation anyway. Probably best to either choose one place, or else make it a new component. If you need another opinion, @crhallberg might be a good person to ask, as he's had more to do with the organization of the project's styles than I have.

Reply:
I can do either but I would rather have @crhallberg point of view, please

Reply:
These files don't impart any different scoping or specificity, so these two blocks will be equivalent in the compiled CSS. Considering that the template is in "search", I think it would be clearest to remove this one in record.less.

Reply:
Thanks